frontend/__tests__/components/FilterPanel.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { render, screen } from '@testing-library/react'
+import FilterPanel from '../../src/components/FilterPanel'
+import '@testing-library/jest-dom'
+
+describe(FilterPanel, () => {
+  it('renders a preferences header', () => {
+    render(<FilterPanel />)
+    expect(screen.getByText('Your Preferences')).toBeVisible()
+  })
+
+  it('displays a filter category for each Dietary Restrictions filter', () => {
+    const DIETARY_RESTRICTIONS = [
```

Comment:
There are the vars in this file that would need to be down-cased as well, if you have time